### ISSUE TITLE
Render supplementary.pdf directly in results

### ISSUE THREAD

**Issue by johnne** [NBISweden/workshop-reproducible-research#17]
The complete Snakefiles (git/, docker/, singularity/, jupyter/) now render in code/ and the output is manually moved to results/ instead of using output_dir as a render argument. This was due to not working in a Singularity container for some reason (the pdf ended up in code/ anyways plus an empty file in results/). This should possibly be fixed.

Potentially this could be done with adjusting the `supplementary_material.Rmd` file with pandoc-arguments:

```yaml
---
title: "Supplementary Material"
author: John Doe, Joan Dough, Jan Doh, Dyon Do
date: "`r format(Sys.time(), '%d %B, %Y')`"
output:
  pdf_document:
    includes:
      in_header: header.tex
    pandoc_args: [
              "--pdf-engine", "pdflatex",
              "--pdf-engine-opt", "-output-directory=results/"
    ]
params:
  counts_file: NA
  multiqc_file: NA
  rulegraph_file: NA
  SRR_IDs: NA
  GSM_IDs: NA
---
```

**Comment by fasterius** [comment]
Where are we on this issue? What is needed to fix?

**Comment by johnne** [comment]
Fixed with [d29ebe1](https://github.com/NBISweden/workshop-reproducible-research/pull/68/commits/d29ebe15cdb92515155ac0294a9b55bdf0bc3889).
